webapp/src/components/searchDialog/searchDialog.tsx

```diff
@@ -36,6 +41,41 @@ const SearchDialog = (props: Props): JSX.Element => {
 
     const emptyResult = results.length === 0 && !isSearching && searchQuery
 
+    const handleUpDownKeyPress = (e: KeyboardEvent) => {
+        if (Utils.isKeyPressed(e, Constants.keyCodes.DOWN)) {
+            e.preventDefault()
+            if (results.length > 0)
+                setSelected(Utils.clamp(selected + 1, 0, results.length - 1))
+        }
+
+        if (Utils.isKeyPressed(e, Constants.keyCodes.UP)) {
+            e.preventDefault()
+            if (results.length > 0)
+                setSelected(Utils.clamp(selected - 1, 0, results.length - 1))
+        }
+    }
+
+    const handleEnterKeyPress = (e: KeyboardEvent) => {
+        if (Utils.isKeyPressed(e, Constants.keyCodes.ENTER) && selected >= 0) {
+            e.preventDefault()
+            props.refs.current[selected].current.click()
+        }
+    }
+
+    useEffect(() => {
+        if (selected >= 0)
+            props.refs.current[selected].current.parentElement.focus()
+
+        document.addEventListener('keydown', handleUpDownKeyPress)
```

Comment:
Again, probably I would prefer to use the hook `useHotkeys` here.

Reply:
Yeah, good call out. Like what I mentioned above, after fighting with native browser/OS shortcuts I did the event listeners natively and didn't consider the hotkey library. I'll see about getting it refactored.

Reply:
Tried locally here too and same issue as above, hotkeys doesn't work in this case because focused on an `<input>`.